src/js/actions/libraries.js

```diff
@@ -41,9 +42,13 @@ define(function (require, exports) {
         locks = require("../locks"),
         layerActions = require("./layers"),
         searchActions = require("./search/libraries"),
+        shapeActions = require("./shapes"),
```

Comment:
While you're in here, can you also sort this require list to make it a bit more readable.